src/ainft.ts

```diff
@@ -33,6 +36,8 @@ export default class AinftJs {
   public textToArt: TextToArt;
   public activity: Activity;
   public eth: Eth;
+  public baseAI: BaseAI;
```

Comment:
What is difference between BaseAI and chatAI?

Reply:
What about baseAi?

Reply:
Done! I renamed baseAI to `baseAi`.
The difference between `baseAi` and `chatAi` is that `baseAi` is an implementation class that manages common functions for AI (config, credit, request, etc..). 
And `chatAi` is an implementation class that processes special requests related to chat. 
This chatAi (sub)class inherits baseAi class.

Reply:
Fyi. The baseAi instance being initialized in the above ainft constructor will be deleted soon.
This line was added temporarily because of the testcode. User will use only the chatAi instance initialized.